lib/src/main/java/com/auth0/android/lock/Lock.java

```diff
@@ -136,16 +136,17 @@ public void onDestroy(@NonNull Context context) {
     }
 
     private void initialize(@NonNull Context context) {
+        LocalBroadcastManager lbm = LocalBroadcastManager.getInstance(context);
+        lbm.unregisterReceiver(this.receiver);
```

Comment:
this is a no-op when the receiver was not previously registered. Is a safe way of making sure we only register this one once, avoiding the same event to be delivered multiple times to the dev provided callback.